src/Task.ts

```diff
@@ -231,6 +245,8 @@ export class Task {
         let doneDate: Moment | null = null;
         let recurrence: Recurrence | null = null;
         let tags: any = [];
+        // Tags that are removed from the end while parsing, but we want to add them back for being part of the description
+        let trailingTags = '';
```

Comment:
Minor observation:

This is called `trailingTags` but I found that it parsed this and retained all the values...

`- [ ] this is a task due 📅 2021-09-12 #inside_tag ⏫ #some/tags_with_underscore ⏳ 2021-12-28 #more_tags`

So maybe `trailing` is a bit confusing?

Reply:
True, "trailing" here actually means "after any description text" and not "after everything", which may not be the best name.